src/utils/daos_hdlr.c

```diff
@@ -2096,143 +2000,59 @@ file_mkdir(struct cmd_args_s *ap, struct file_dfs *file_dfs,
 			rc = errno;
 		}
 	} else if (file_dfs->type == DAOS) {
-		rc = mkdir_dfs(ap, file_dfs, dir, mode);
+		rc = dfs_sys_mkdir(file_dfs->dfs_sys, dir, *mode, 0);
 		if (rc != 0) {
 			/* mkdir_dfs already prints error */
```

Comment:
This isn't true about dfs_sys_mkdir, which doesn't print any errors

Reply:
@daltonbohning yeah sorry this was an old comment that got left here.